src/modules/Elsa.Workflows.Core/Extensions/BackgroundActivityExecutionContextExtensions.cs

```diff
@@ -45,6 +45,22 @@ public static void SetBackgroundOutcomes(this ActivityExecutionContext activityE
         return activityExecutionContext.GetProperty<IEnumerable<string>>("BackgroundOutcomes");
     }
     
+    /// <summary>
+    /// Sets the background outcomes.
+    /// </summary>
+    public static void SetBackgroundCompletion(this ActivityExecutionContext activityExecutionContext)
+    {
+        activityExecutionContext.SetProperty("BackgroundCompletion", true);
+    }
+    
+    /// <summary>
+    /// Gets the background outcomes.
+    /// </summary>
+    public static bool? GetBackgroundCompleted(this ActivityExecutionContext activityExecutionContext)
```

Comment:
GetBackgroundCompletion

Reply:
You're right, that sounds more correct 👍🏻